relay-general/src/store/normalize/span/description.rs

```diff
@@ -36,6 +38,13 @@ static SQL_NORMALIZER_REGEX: Lazy<Regex> = Lazy::new(|| {
     .unwrap()
 });
 
+/// Removes extra whitespace and newlines.
+static SQL_WHITESPACE: Lazy<Regex> = Lazy::new(|| Regex::new(r"(\s*\n\s*)|(\s\s+)").unwrap());
+
+/// Removes whitespace around parentheses.
+static SQL_PARENS: Lazy<Regex> =
+    Lazy::new(|| Regex::new(r"((?P<pre>\()\s+)|(\s+(?P<post>\)))").unwrap());
```

Comment:
Q: do we need the capture group names?

Reply:
`scrub_sql_queries` uses them for replacement, so I would say yes.